### PR TITLE
Fixes booze-o-mats not having booze

### PR DESCRIPTION
**Pull request by Anonmare**
[Changelogs]:Alcohol is Humanity's friend, and I will not abandoning a friend

:cl: 
fix: Booze-o-mats have beer
/:cl:

[why]: Qutoes from the Booze-o-mat: ";Care for a nice, cold beer?", ";Have a beer!", ";Beer is good for you!" and ";Man loves beer."

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Comment by Anonmare** [comment]
This is clearly a fix, Booze-o-mats are obviously intended to have beer, don't deny me my GBPs

File: code/game/objects/items/vending_items.dm [hunk 1]
```diff
@@ -37,8 +37,8 @@
 /obj/item/vending_refill/boozeomat
 	machine_name = "Booze-O-Mat"
 	icon_state = "refill_booze"
-	charges = list(54, 4, 0)//of 159 standard, 12 contraband
-	init_charges = list(54, 4, 0)
+	charges = list(60, 4, 0)//of 168 standard, 12 contraband
```
Comment (ShizCalev) [inline code/game/objects/items/vending_items.dm]: This number is wrong.
Should be 58, to refill for a total of 174.

File: code/game/objects/items/vending_items.dm [hunk 2]
```diff
@@ -37,8 +37,8 @@
 /obj/item/vending_refill/boozeomat
 	machine_name = "Booze-O-Mat"
 	icon_state = "refill_booze"
-	charges = list(54, 4, 0)//of 159 standard, 12 contraband
-	init_charges = list(54, 4, 0)
+	charges = list(60, 4, 0)//of 168 standard, 12 contraband
+	init_charges = list(60, 4, 0)
```
Comment (ShizCalev) [inline code/game/objects/items/vending_items.dm]: update it here too.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/objects/items/vending_items.dm (hunk 1 above)
Comment (Anonmare) [reply code/game/objects/items/vending_items.dm]: I counter, there are 180 total items in total

File: code/game/objects/items/vending_items.dm [hunk 3]
```diff
@@ -37,8 +37,8 @@
 /obj/item/vending_refill/boozeomat
 	machine_name = "Booze-O-Mat"
 	icon_state = "refill_booze"
-	charges = list(54, 4, 0)//of 159 standard, 12 contraband
-	init_charges = list(54, 4, 0)
+	charges = list(58, 4, 0)//of 162 standard, 12 contraband
```
Comment (ShizCalev) [inline code/game/objects/items/vending_items.dm]: it's 174.

image

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**